server.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const fastify = require('fastify')({ logger: true })
+
+const postItem = require('./handlers/postItem');
+const getAllItems = require('./handlers/getAllItems');
+const getCategories = require('./handlers/getCategories')
+
+fastify.post('/postItem', postItem);
+fastify.get('/', getAllItems);
+fastify.get('/categories', getCategories);
+
+const start = async () => {
+  try {
+    await fastify.listen(3000)
+    fastify.log.info(`server listening on ${fastify.server.address().port}`)
+  } catch (err) {
```

Comment:
In Fastify you have an option to create a custom error handler that can gracefully send all kinds of errors so you can just throw them without catching everywhere. Try to check this: [https://www.fastify.io/docs/master/Errors/#errors-in-fastify](https://www.fastify.io/docs/master/Errors/#errors-in-fastify)